doc/radosgw/notifications.rst

```diff
@@ -104,16 +104,17 @@ Notification Performance Statistics
 -----------------------------------
 
 - ``persistent_topic_size``: queue size in bytes. 
-- ``pubsub_event_triggered``: a running counter of events that have at least one topic associated with them
+- ``persistent_topic_len``: shows how many notifications are currently waiting
```

Comment:
i dont think we have topic length in reef. why did you add it?

Reply:
we also dont have `persistent_topic_size` in reef

Reply:
> i dont think we have topic length in reef. why did you add it?

This whole set of PRs is made in advance of a near-future PR that will cater to each of the releases' requirements. If you prefer, I can alter this PR, but it will then not map onto the history of the development of the documentation in a clean way.

Reply:
Okay, I've fixed it up in this PR. Let me know if it's ready to merge.